lib/game.js

```diff
@@ -0,0 +1,59 @@
+const Board = require('./board.js');
+const Player = require('./player.js');
+
+module.exports = class Game {
+  constructor(playerX, playerO) {
+    this.board = new Board();
+    this.playerX = playerX;
+    this.playerO = playerO;
+    this.currentPlayer = playerX;
+  }
+
+  play() {
+    if (!this.isOver()) {
+      this.playTurn();
+    } else {
+      this.endGame();
+    }
+  }
+
+  isOver() {
+    return this.board.hasWin() || this.board.hasTie();
+  }
+
+  startGame() {
+    console.log("Welcome to Tic Tac Toe");
+  }
+
+  async playTurn() {
+    this.printBoard();
+    console.log(`Player ${this.currentPlayer.getSymbol()}, it's your turn.`);
+
+    let input = await this.currentPlayer.getInput();
+
+    this.markBoard(input.move);
```

Comment:
Can you abstract this notion of input?

```
let move = await this.currentPlayer.move();
this.markBoard(move);
```

Maybe you could reuse the same variable `move` for each turn instead of reassigning a new variable, but to be honest it's not likely to be a problem :)